mongo/document/validation_error.go

```diff
@@ -0,0 +1,43 @@
+package document
+
+import "bytes"
+
+type ValidationError struct {
+	Errors map[string][]string `json:"errors"`
+}
+
+func NewValidationError() *ValidationError {
```

Comment:
Don't we want to add an other constructor which takes arugments so that we could write `return NewValidationError("my-field", "errmsg")`?

Reply:
We could but i don't see what could be the usage

Reply:
When there is only one field to validate?